src/Domain/DoctrineDonationRepository.php

```diff
@@ -185,6 +185,9 @@ public function findWithExpiredMatching(\DateTimeImmutable $now): array
             ->innerJoin('d.fundingWithdrawals', 'fw')
             ->where('d.donationStatus IN (:expireWithStatuses)')
             ->andWhere('d.createdAt < :expireBefore')
+            // First of a regular giving series is Pending during 3DS. If we ever make the timeout for
+            // that longer than the timeout for matching, we still want to ensure matching can't be
+            // lost while 3DS is in progress.
```

Comment:
OK - somewhat duplicates the comment on this method in the interface. That is one disadvantage of having interfaces, can be harder to see the docs.

https://github.com/thebiggive/matchbot/blob/73d6c83e0bf5f136467b400fbb87519ec6bf7270/src/Domain/DonationRepository.php#L60

Reply:
Yeah, I think I probably wrote this before that was merged in, was just trying to include some uncommitted bits I had in-memory while I somewhat remembered why they were there. Can get rid or combine the two if you think it's better in the doc block?

Reply:
I don't have a strong feeling, could be worth combining but not a blocker.